setup.py

```diff
@@ -42,7 +42,7 @@
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
         'Operating System :: POSIX :: Linux',
-        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.6',
```

Comment:
Don't we support 3.5? Just to make it clear.

Reply:
no, 3.5 is missing quite a few features. among other things the f-string. and some methods for testing, i cannot recall the exact names.

but 3.6 is a much more capable python3 than 3.5.

Reply:
thanks